drivers/mipi_dsi/Kconfig.stm32

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2023 bytes at work AG
+# SPDX-License-Identifier: Apache-2.0
+
+config MIPI_DSI_STM32
+	bool "STM32 MIPI-DSI Host Controller"
+	default y
+	depends on DT_HAS_ST_STM32_MIPI_DSI_ENABLED
+	select USE_STM32_HAL_DSI
+	select STM32_LTDC
+	help
+	  STM32 MIPI DSI controller driver
+
+if MIPI_DSI_STM32
+
+config MIPI_DSI_STM32_INIT_PRIORITY
+	int "Initialization priority"
+	default MIPI_DSI_INIT_PRIORITY
+	help
+	  MIPI-DSI STM32 Host Controllers initialization priority.
```

Comment:
why isn't default `MIPI_DSI_INIT_PRIORITY` sufficient?

Reply:
It is, thanks.